docs/detailed-usage.md

```diff
@@ -97,42 +97,30 @@ Channel status can be checked with:
 state.fsm_state
 ```
 
-## VM usage
+## Contracts usage
 
-Initial implementation of the AEVM (Aeternity Virtual Machine) that contains all of the functionalities of the EVM (Ethereum Virtual Vachine).
+With every created contract, there is also a normal account with balance, to which we can perform spend transactions. All charges are being subtracted from the account of the creator/caller, before the call of the contract, together with gas*gas_price aeons. Each create/call will use up specific amount of gas, up to the maximum provided. Any remaining portion of gas will be refunded to the caller's account. If the initial call fails, amount and deposit are being returned to the creator.
 
-To run the VM you can use the following command:
-```
-Aevm.loop(
```

Comment:
This would still be good to be included, if it still works like this

Reply:
It is now more difficult to set it up, that's why I decided to remove it

Reply:
Is it still possible to run contracts without using them on-chain? As debug method?

Reply:
At the moment, no.